### ISSUE TITLE
Implement simulation for `MsgCreateClient`

### ISSUE THREAD

**Issue by giunatale** [tendermint/spn#705]
I've noticed the implementation for this simulation is empty, and contains a `TODO`.

We should eventually implement this message simulation.

**Comment by lubtd** [comment]
Yes although this message simulation will be more complex to implement since the completion depends in some way on external event: fetching the consensus state and validator set of launched chain and parsing the values with the values in the `launch` module.
This will require to craft consensus state and validator state from `GenesisValidator` that have been set in the `launch` module for specific launch ID
